packages/esm-patient-alerts-app/src/store.ts

```diff
@@ -1,18 +1,26 @@
 import { Store } from 'unistore';
```

Comment:
@denniskigen would you mind commenting on the reasons for using unistore in this case? It will help others better understand down the line.

Reply:
Sure. So we've got the notification panel in the navbar that is part of the primary navigation microfrontend. We've also got this patient-alerts microfrontend. I want the notification panel to respond to changes in the alerts microfrontend. When alerts are available, we want the notification icon in the navbar to change to `NotificationNew20`. I use unistore as a global state management tool to store state related to the patient alerts microfrontend. With it, I'm able to pass around the state between those two different microfrontends.